src/value.c

```diff
@@ -259,6 +273,36 @@ const char *SIType_ToString(SIType t) {
 	}
 }
 
+void SIType_ToMultipleTypeString(SIType t, char **buf, size_t *bufferLen, size_t *bytesWritten) {
+	SIType remainingTypes = t;
+	SIType currentType;
+	// Allocate space
+	// Worst case: Len(SIType names) + 19*Len(", ") + Len("Or") = 177 + 38 + 2 = 217
+	if(*bufferLen - *bytesWritten < 256) {
+		*bufferLen += 256;
+		*buf = rm_realloc(*buf, sizeof(char) * *bufferLen);
+	}
+	// Iterate over the possible SITypes
+	for(int i = 0; i < 18; i ++) {
+		currentType = (1 << i);
+		if(t & currentType) {
+			if(t == remainingTypes) {
+				remainingTypes &= (~currentType);
+			} 
```

Comment:
what is this?

Reply:
It was to detect the first type detected, now the code was changed to count the number of types detected.